src/Symfony/Component/Translation/Bridge/Crowdin/composer.json

```
@@ -0,0 +1,37 @@
+{
+    "name": "symfony/crowdin-translation",
```

```suggestion
    "name": "symfony/crowdin-translation-provider",
```

but depends on decision in https://github.com/symfony/symfony/pull/40944/files#r620020546